ui/src/containers/NodeVolumes.js

```diff
@@ -40,8 +39,12 @@ import {
 const VolumeTable = styled.div`
   flex-grow: 1;
   margin-top: ${padding.small};
-  .sc-table-column-cell-container-action {
-    justify-content: center;
+  /*solve the tooltip display issue*/
```

Comment:
`//` instead of `/* … */` for single line comment, no?

Also, you should describe here or in the commit message what is the issue your worked around.

Reply:
/*...*/  is the way to give commend in CSS ^^
But yes, for sure it's good to describe what's the issue. Will add it.
Thanks for the review!